assets/stylesheets/modules/_forms.scss

```diff
@@ -89,12 +89,12 @@ body:not(:-moz-handler-blocked) fieldset {
         display: block;
     }
 }
-.js-on .field-panel.is-collapsed {
-    .field-panel__intro,
-    .field-panel__fields {
-        display: none;
-    }
-}
+// .js-on .field-panel.is-collapsed {
```

Comment:
No, I just commented it out to uncollapse the panels for testing. This branch is just a snapshot of where I got to so we can chat through what stuff is a bit temporary.